### PR TITLE
fix error, add response statusCode, sendKafka

### PR DESCRIPTION
**Pull request by vqtinh2111**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-version: '3.7'
+version: '3'
```
Comment (trongtb88) [inline docker-compose.yml:1]: Why we not use version 3.7. I used this version and work fine

Comment (vqtinh2111) [reply docker-compose.yml:1]: oh sorry, my bad. my docker-compose version can't run version "3.7" in docker-compose.yaml file, so i changed it to 3. I should revert it before push.

File: userservice/src/main/java/com/dragon88/userservice/dto/UserReserve.java [hunk 2]
```diff
@@ -2,15 +2,15 @@
...
 import lombok.Setter;
 
 import java.util.List;
 
-@Getter
-@Setter
+@Data
 public class UserReserve {
-    private int id;
+    private Integer id;
```
Comment (trongtb88) [inline userservice/src/main/java/com/dragon88/userservice/dto/UserReserve.java:13]: why we need wrap class here ?

**Review by trongtb88 — APPROVED** [review]
g2g

**Pull request merged**